gz-grant.html

```diff
@@ -119,118 +134,178 @@
         attached: function() {
           console.log('gz-grant widget attached!')
         },
+
         grant: function() {
```

Comment:
For gz-grant's internal config gui, I'm adding `_tapGrant` and `_tapRevoke`, which internally call `grant` and `revoke`